api/Node.json

```diff
@@ -1154,6 +1154,1019 @@
             "deprecated": false
           }
         }
+      },
+      "appendChild": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Node/appendChild",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "1"
+            },
+            "firefox_android": {
+              "version_added": "4"
+            },
+            "ie": {
+              "version_added": true
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": true
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "cloneNode": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Node/cloneNode",
+          "support": {
+            "webview_android": {
+              "version_added": true,
+              "notes": "<code>deep</code> optional parameter, default value is <code>false</code>"
+            },
+            "chrome": {
+              "version_added": true,
+              "notes": "<code>deep</code> optional parameter, default value is <code>false</code>"
+            },
+            "chrome_android": {
+              "version_added": true,
+              "notes": "<code>deep</code> optional parameter, default value is <code>false</code>"
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": [
+              {
+                "version_added": true
+              },
+              {
+                "version_added": "13",
+                "notes": "<code>deep</code> as an optional parameter"
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": true
+              },
+              {
+                "version_added": "14",
+                "notes": "<code>deep</code> as an optional parameter"
```

Comment:
ditto (also, this was in Firefox for Android 13 and not 14).

Reply:
Firefox android version 13 gets rejected by the linter and build test.
I added the versions in [this pull request](https://github.com/mdn/browser-compat-data/pull/1715). Will change if/when merged.